src/TesApi.Web/BatchScheduler.cs

```diff
@@ -852,7 +1006,7 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(TesTask task, bool
                 // This also requires that the main task runs inside a container. So we run the "docker" container that in turn runs other containers.
                 // If the executor image is public, there is no need for pool ContainerConfiguration and task can run normally, without being wrapped in a docker container.
                 // Volume mapping for docker.sock below allows the docker client in the container to access host's docker daemon.
-                var containerRunOptions = $"--rm -v /var/run/docker.sock:/var/run/docker.sock -v $AZ_BATCH_TASK_WORKING_DIR:$AZ_BATCH_TASK_WORKING_DIR ";
+                var containerRunOptions = $"--rm -v /var/run/docker.sock:/var/run/docker.sock -v $AZ_BATCH_NODE_ROOT_DIR:$AZ_BATCH_NODE_ROOT_DIR ";
```

Comment:
why wouldn't we use the working directory?

Reply:
We reach above the working directory in order to clear out debris left over from previous tasks on the same node.